app/Project.swift

```diff
@@ -50,6 +54,10 @@ let project = Project(
                     "DEVELOPMENT_TEAM": "U6LC622NKF",
                     "CODE_SIGN_STYLE": "Automatic",
                     "CODE_SIGN_IDENTITY": "Apple Development",
+                ],
+                release: [
+                    "OTHER_CODE_SIGN_FLAGS": "--timestamp --deep",
```

Comment:
Why are these flags needed? Worth adding a note about them above the line.

Reply:
These are required for the subsequent notarization.